test/cli/web-trace-file-non-ascii/web-trace-file-non-ascii.rb

```diff
@@ -0,0 +1,60 @@
+# typed: true
+
+class HundredValEnum < T::Enum
+  enums do
+    Val001 = new
+    Val002 = new
+    Val003 = new
+    Val004 = new
+    Val005 = new
+    Val006 = new
+    Val007 = new
+    Val008 = new
+    Val009 = new
+    Val010 = new
+    Val011 = new
+    Val012 = new
+    Val013 = new
+    Val014 = new
+    Val015 = new
+    Val016 = new
+    Val017 = new
+    Val018 = new
+    Val019 = new
+    Val020 = new
+  end
+
+  # This string literal contains U+2019, which is a non-ascii character.
+  # We used to essentially have something like this at Stripe,
+  # and it would crash us when trying to serialize a web trace file
+  # (because the method name was in one of the args)
+  it '’' do
```

Comment:
I verified that this test failed here:

https://buildkite.com/sorbet/sorbet/builds/9847#1e9ff4d8-51c2-418c-bcab-eca775822355